templates/elasticsearch.yml.j2

```diff
@@ -8,7 +8,7 @@ cluster.name: elasticsearch
 {% endif %}
 
 {% if es_config['node.name'] is not defined %}
-node.name: {{inventory_hostname}}-{{es_instance_name}}
+node.name: {{inventory_hostname}}{{' - ' + es_instance_name if es_use_shared_fs else ''}}
```

Comment:
Are the extra spaces around ` - ` intentional? This is causing the tests to fail because the node changed `localhost-node1` to `localhost - node1`